root-expo-object.md

```diff
@@ -0,0 +1,79 @@
+# Migrating from the root-level `expo` object in the Expo config
```

Comment:
```suggestion
# Migrating from the root-level `expo` object in the app config
```

For consistency, let's stick using "app config" instead of Expo config since this is what we also use in our docs: https://docs.expo.dev/workflow/configuration/